code/modules/clothing/outfits/standard.dm

```diff
@@ -244,15 +244,15 @@
 	back = /obj/item/storage/backpack/satchel/leather
 	id = /obj/item/card/id
 
-/datum/outfit/spec_ops/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+/datum/outfit/admiral/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
 	if(visualsOnly)
 		return
 
 	var/obj/item/card/id/W = H.wear_id
 	W.icon_state = "centcom"
 	W.access = get_all_accesses()
-	W.access += get_centcom_access("Special Ops Officer")
-	W.assignment = "Special Ops Officer"
+	W.access += get_centcom_access("Admiral")
```

Comment:
This won't work because you haven't said what access an admiral gets in the `get_centcom_access` proc.

Reply:
Admiral gets all centcom access it's in the access files.

Reply:
No, special ops officer gets all centcom access. You need to change the name in the get_centcom_access proc

Reply:
nevermind im blind, merging